src/hooks/game/useBalanceGameCreation.ts

```diff
@@ -73,12 +123,30 @@ export const useBalanceGameCreation = (
     );
   };
 
+  const handleOptionChange =
+    (optionType: 'A' | 'B') => (event: React.ChangeEvent<HTMLInputElement>) => {
+      updateOption(currentStage, optionType, { name: event.target.value });
+    };
+
+  const handleDescriptionChange = (
+    optionType: 'A' | 'B',
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    const { value } = event.target;
+    updateOption(currentStage, optionType, { description: value });
+  };
```

Comment:
두 함수가 비슷한 작업을 수행하고 있는데, 통합하여 props로 전달받은 후 상태를 업데이트 하는 방식은 어떨까요? 

```suggestion
  
  const handleOptionUpdate =
    (optionType: 'A' | 'B', field: 'name' | 'description') =>
    (event: React.ChangeEvent<HTMLInputElement>) => {
        const { value } = event.target;
        updateOption(currentStage, optionType, { [field]: value });
  };
```

Reply:
오 같은 핸들러로 처리하는 방법도 있었네요! 시도해보고 추가 코멘트 남기겠습니다!

Reply:
[2523713](https://github.com/JECT-Study/PICK-O-Client/pull/250/commits/2523713afbd1dcbc5b310fdd39eae65f4b3b7e11) 에서 처리했습니다!